pyproject.toml

```diff
@@ -43,6 +43,9 @@ exclude=["tests"]
 [tool.pytest.ini_options]
 addopts = "--verbose"
 markers = ["slow: marks tests as slow"]
+pythonpath = [
```

Comment:
I would prefer to not change a default affecting the whole pytest execution. Maybe you could use importlib to have direct access to the "external" module/function you want to test

Reply:
That's only for unit tests, but OK, I will see what could be done about it.